### DISCUSSION TITLE
player settings help me

### DISCUSSION THREAD

**Discussion by oderece** [ant-media/Ant-Media-Server#6220] [General]
Hello; I created a new onvif camera. I added the camera and the broadcast was active but it turned green. When I try to preview it in the player, it gives an error.

How can I adjust player settings?

I can only watch when I save it as mp4. But I can't watch live recordings.
thanks

**Comment by burak-58** [comment]
Hi @oderece,
Thanks for your entry. Let me convert this to a discussion first. You may get an answer earlier. After discussion, we may convert this to issue again if you can't get a solution there.

**Comment by USAMAWIZARD** [comment]
Hi @oderece ,
what kind of error are you getting when trying to play the recording files ? 

can you please share a screenshot of error message. what kind of codes are you using on the camera side and what are the encoding parameters ? are you able to preview the live streams ??
